.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+      "no-redeclare": "off",
+      "no-unused-vars": "off",
+      "no-undef": "off",
+      "no-mixed-spaces-and-tabs": "off",
+      "no-extra-semi": "off",
+      "no-empty": "off",
+      "no-console": "off"
+    }
+}
```

Comment:
Switched from `semistandard` to `eslint` since the codebase violates a lot of semistandard in its current state.
Still had to turn off some eslint rules. I'd say the first four are ones we'd really want to turn back on once the offending code is cleaned up.

Reply:
Cleaned the code up and turned those switches on: https://github.com/networked-aframe/networked-aframe/pull/91